hutch_python/load_conf.py

```diff
@@ -16,6 +16,8 @@
 from bluesky.callbacks.best_effort import BestEffortCallback
 from bluesky.callbacks.core import LiveTable
 from bluesky.callbacks.mpl_plotting import initialize_qt_teleporter
+from event_model import RunRouter
```

Comment:
I guess I'll finally look up what RunRouter is- code here looks good (the beef of the code is in the other PR)